docs/API/LorisRESTAPI_v0.0.3.md

```diff
@@ -728,3 +728,63 @@ The JSON object is of the form:
     "Value" : string
 }
 ```
+
+# 5.0 DICOM Data
+
+Like the imaging data, the DICOM data mostly lives in the `/candidates/$CandID/$Visit` 
+portion of the REST API namespace, but is defined in a separate section of this 
+document for clarity purposes.
+
+## 5.1 Candidate DICOMs
+```
+GET /candidates/$CandID/$Visit/dicoms
+```
+
+A GET request to `/candidates/$CandID/$Visit/dicoms` will return a JSON object of
+all the raw DICOM data which have been acquired for that visit. It will return an 
+object of the form:
+
+```js
+{
+    "Meta" : {
+        "CandID" : $CandID,
+        "Visit" : $VisitLabel,
+    },
+    "DicomTars" : 
+        [{
+        "Tarname" : "DCM_yyyy-mm-dd_ImagingUpload-hh-mm-abc123.tar",
+        "SeriesInfo" :
+            [{
+            "SeriesDescription" : "MPRAGE_ipat2",
+            "SeriesNumber" : "2",
+            "EchoTime" : "2.98",
+            "SeriesUID" : "1.2.3.4.1107",
+            },
+            {
+            "SeriesDescription" : "BOLD Resting State",
+            "SeriesNumber" : "5",
+            "EchoTime" : "30",
+            "SeriesUID" : "3.4.5.6.1507",
+            }],
+        "Tarname" : "DCM_yyyy-mm-dd_ImagingUpload-hh-mm-def456.tar",
+        "SeriesInfo" :
+            [{
+            "SeriesDescription" : "MPRAGE_ipat2",
+            "SeriesNumber" : "2",
+            "EchoTime" : "2.98",
+            "SeriesUID" : "1.7.8.9.1296",
+            }],
+    }],    
+}
+```
+
+## 5.2 Tar Level Data
+```
+GET /candidates/$CandID/$VisitLabel/dicoms/$Tarname
+```
+
+Returns a `tar` file which contains a `.meta` and a `.log` text files, 
```

Comment:
OK, I found out why it was not working, the SessionID was NULL for that archive. Maybe in the query in line 104 of `visits/Dicoms.php`, there should be a join to the session table and candidate table, as it is done for the `dicoms/Dicom.php` script? Technically, we should not be able to view DICOM archives that you cannot download.

Reply:
but if your session ID is NULL< isn;t it because you ran dicomTar but not tarchiveLoader?
in that case, you would still have a DICOM that should be downloadable, non?
the path will not have the year in it, but the script does not care as it simply appends your config settings to where the .tar files are (usually in the tarchive/ folder) to the rchiveLocation in the tarchive table (with or without a year). 
To me, as long as there is metadata, this means the tarchive_series, tarchive_files have been populated, meaning there is a .tar somewhere that should be downloadable...

Reply:
I agree, we want to be able to download them but for some reason, it does not work.

Here is my archiveLocation:
![screen shot 2018-05-10 at 3 43 38 pm](https://user-images.githubusercontent.com/1402456/39890742-aff3993e-5469-11e8-9a08-6d8ff5c88c60.png)

Here is what I get when I try to download it:
![screen shot 2018-05-10 at 3 43 46 pm](https://user-images.githubusercontent.com/1402456/39890762-be7fc16c-5469-11e8-990b-be384f88e59b.png)

Weird... I don't see why this would not work... @MounaSafiHarab Am I missing something?